### ISSUE TITLE
Entering an old GND 

### ISSUE THREAD

**Issue by rettinghaus** [saw-leipzig/basic.app#22]

**Comment by rettinghaus** [comment]
When adding manually an URI with redirect like `http://d-nb.info/gnd/10364445-3` it should return directly the latest ID. (Perhaps with a warning popping up.)

**Comment by ukretschmer** [comment]
Good point. We may also switch the data backend from entityfacts to lobid to be able to handle old IDs explicitly and not by interpreting redirects. This would be possible, because lobid also delivers `oldAuthorityNumber`.

Comparable JSONs of mentioned example

- *Lobid:* [http://lobid.org/gnd/1073054446.json](http://lobid.org/gnd/1073054446.json)
- *Entity facts:* [http://hub.culturegraph.org/entityfacts/1073054446](http://hub.culturegraph.org/entityfacts/1073054446)

**Comment by rettinghaus** [comment]
Using lobid sounds reasonable!
